src/main/java/net/sf/jabref/gui/FileExtensions.java

```diff
@@ -0,0 +1,61 @@
+package net.sf.jabref.gui;
+
+public enum FileExtensions {
+
+    //importannt: No dot before the extension!
+    BIBTEX_DB(
+            "BibTeX Database", "bib"),
+    BIBTEXML(
+            "BibTexMl, bibx"),
+    AUX(
+            "Aux file", "aux"),
```

Comment:
AUX file? I actually like "Aux" more but we recently had this discussion about uniform file type names